ybacklight

```diff
@@ -64,7 +64,7 @@ sub get_percentage {
     my ($driver) = @_;
     my $bri = get_numeric_attribute($driver, "brightness");
     my $bri_max = get_numeric_attribute($driver, "max_brightness");
-    my $pct = int($bri/$bri_max*100);
+    my $pct = ($bri/$bri_max*100);
     say "Current brightness $bri/$bri_max ($pct%)" if $verbose;
```

Comment:
Please round the (floating-point) percentage here before printing (`say "... (@{[int $pct]}%)" if $verbose`)